pegasus/sites.v3/code.org/views/header.haml

```diff
@@ -20,24 +20,3 @@
     #loc.desktop-feature
       -if banner = @header['banner']
         =view banner
-
-:javascript
-  var headerSetup = false;
-
-  $ = jQuery;
-  $(document).ready(function() {
-    if (headerSetup) {
-      return;
-    }
-    headerSetup = true;
-
-    $.ajax({
-      type: "GET",
-      url: '/dashboardapi/user_menu',
-      //data: "id=" + id, // appears as $_GET['id'] @ ur backend side
-      success: function(data) {
-        // data is ur summary
-        $('#sign_in_or_user').html(data);
-      }
-    });
-  });
```

Comment:
The code to ajax-load the sign-in button is now located in [`hamburger.js`](https://github.com/code-dot-org/code-dot-org/blob/60c8773d0121effb93a1936bbc222a2e76ca30c8/apps/src/hamburger/hamburger.js#L34-L40) following the header redesign, so this duplicate ajax call can be removed.